Proposition för Ändring av Rättigheter för Stödmedlem och Gästmedlem - SM#3/SM#4 2025 - 2025-04-22/2025-09-30 (#44)
insektionen/styrdokument: stadgar/eng/stadgar.md

Reviewer: This needs to be updated with the new guest members at SM...

§4.3.1 A supporting member is a member of THS that has chosen to support the chapter.

> §4.3.2 A supporting member has the same rights as a regular member, except for the right to vote at chapter meetings and in the THS union council elections, as well as eligibility for positions in the union council and the board.

Reviewer: ```suggestion
§4.3.2 A supporting member has the same rights as a regular member, except for the right to vote at chapter meetings and in the THS union council elections, as well as eligibility for positions in the THS union council and the chapter board.
```

> §4.3.3 A supporting member have the same responsibilities as a regular member.

Reviewer: ```suggestion
§4.3.3 A supporting member has the same responsibilities as a regular member.
```
Not really for this proposition, but fix grammar :)

> ### §4.4 Honorary member

Reviewer: Good find :D

§4.4.5 An honorary member that has inflicted harm upon the chapter may be expulsed after a decision has been made with a qualified majority on a chapter meeting.

### §4.5 Guest Member

> §4.5.1 A guest member is a member of THS but does not need to study a program mapped to the chapter.

Reviewer: ```suggestion
§4.5.1 A guest member is a member of THS who does not study a program mapped to the chapter.
```
Update to be the same wording as supportive member.

§4.5.2 A guest member is appointed for the period of one year by a board meeting with a qualified majority.

> A request to be appointed as a guest member should be sent to the board by the person itself at least 4 days before the board meeting.

Reviewer: ```suggestion
§4.5.2 A guest member is appointed by a board meeting with a qualified majority for a period of one year.
A request to be appointed as a guest member must be sent to the board by the person themselves no later than four (4) days before the board meeting.
```
Better wording
